extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -44,12 +44,19 @@ export class VisibilityModel {
      */
     this.spec_ = {
       visiblePercentageMin: Number(spec['visiblePercentageMin']) / 100 || 0,
-      visiblePercentageMax: Number(spec['visiblePercentageMax']) / 100 || 1,
+      visiblePercentageMax: Number(spec['visiblePercentageMax']) / 100,
       totalTimeMin: Number(spec['totalTimeMin']) || 0,
       totalTimeMax: Number(spec['totalTimeMax']) || Infinity,
       continuousTimeMin: Number(spec['continuousTimeMin']) || 0,
       continuousTimeMax: Number(spec['continuousTimeMax']) || Infinity,
     };
+    // If visiblePercentageMax was not specified, assume 100% (but do allow
+    // 0% to have been specified)
+    if (spec['visiblePercentageMax'] === null ||
```

Comment:
it might be clear to just write:

```
if (spec['visiblePercentageMax'].trim() ==='0') {
  this.spec_.visiblePercentageMax = 0;
} else {
  this.spec_.visiblePercentageMax = Number(spec['visiblePercentageMax']) / 100 || 1;
}
```